glotaran/builtin/io/csv/__init__.py

```diff
@@ -0,0 +1 @@
+from . import csv
```

Comment:
Removing this leads to that the csv reader gets not registered, with the other packeges an empty `__init__` is fine...

Reply:
This can be fixed by being more explicit with the entypoints, eg see this solution https://github.com/s-weigand/pyglotaran/commit/264e82cabdb23358af85ef29b1db7745954b5f3c
I can cherrypick it over if you want or make a PR on your branch.